cg-producer/pycg_producer/producer.py

```diff
@@ -275,6 +268,11 @@ def _get_lines_of_code(self, files_list):
         return res
 
 
+    def _get_max_rss(cmd):
```

Comment:
There is a `self` parameter missing here - this function is never called.

Reply:
Sorry for this... fixed on e3f3f41